app/upgrades/v5_0/upgrade.go

```diff
@@ -1,4 +1,5 @@
-package v5
+//nolint:revive
+package v5_0
```

Comment:
Using an underscore in the package name throws a lint error but I think this should be an exception.